### ISSUE TITLE
Upgrade to Elm 0.19

### ISSUE THREAD

**Issue by declension** [declension/elm-obj-loader#3]
Hi. Thanks for this package, keen to get it working (with Elm 0.19)! Not sure what the particular reasons are, but:

```bash
$ elm install Zinggi/elm-obj-loader
```
produces:
```
The following packages do not work with Elm 0.19.0 right now:

    Zinggi/elm-obj-loader
```

Would be happy to help (where I can, not an Elm *or* WebGL expert...)

**Comment by declension** [comment]
Just tried `elm-upgrade`, it did well, but:

```
WARNING! 2 of your dependencies have not yet been upgraded to
support Elm 0.19.
  - https://github.com/Bogdanp/elm-combine
  - https://github.com/Skinney/elm-array-exploration
```

**Comment by Zinggi** [comment]
Hey there.
I haven't had the time to upgrade this for elm 0.19 and I probably wont have it any time soon.

If your interested in getting this upgraded, I can give you some support:

 * elm-array-exploration: Will not be needed in 0.19, just replace it by the standard `Array` in the core library. They are the same now.
 * elm-combine: This was the parsing library I used. It hasn't been upgraded and it never will, as it used many custom operators which are no longer definable for package authors in 0.19. It would have to be replaced with [elm/parser](https://package.elm-lang.org/packages/elm/parser/latest/). Luckily I have a branch with an implementation using a very early version of `elm/parser` (https://github.com/Zinggi/elm-obj-loader/tree/progressBar)

**Comment by declension** [comment]
Thanks for the pointers. I actually started down this path on Friday.

> elm-array-exploration: Will not be needed in 0.19, just replace it by the standard Array in the core library. They are the same now.

Yep did that, no problem

> elm-combine: This was the parsing library I used. It hasn't been upgraded and it never will, as it used many custom operators which are no longer definable for package authors in 0.19.

There's [a fork of `elm-combine` which has 0.19 support](https://github.com/andre-dietrich/parser-combinators), so I started using that. Obviously without all the custom operators it was a _lot_ of work rewriting the `elm-obj-loader` parser but I _think_ it's mostly there now. Of course, maybe the now-native parser was the better way to go, agh. I'll see.

I've left the `elm/http` on version 1.0, too much work to do that too :)

Are there any tests to help me prove it's working though? When I tried compiling the examples, they have _further_ 0.19 compatibility issues so I'm not sure if those would be that useful (in time).

Thanks!
